bots.php

```diff
@@ -70,8 +70,10 @@ function useronline_get_bots() {
 		'Radian6' => 'radian6',
 		'RadioUserland' => 'userland',
 		'Moz' => 'rogerbot',
+                'Rogerbot' => 'rogerbot',  
```

Comment:
Can we fix the alignment?

Reply:
Yes of course. You can do that. Thanks.

Reply:
Can uyoufixed it in the PR?

Reply:
Done!

Reply:
You remove the bot rather than fix the spacing?

Reply:
Yes, i remove the bot because i see that it is listed twice.